RecoTracker/MeasurementDet/plugins/MeasurementTrackerESProducer.cc

```diff
@@ -159,6 +158,9 @@ MeasurementTrackerESProducer::MeasurementTrackerESProducer(const edm::ParameterS
     phase2TrackerCPEToken_ = c.consumes(edm::ESInputTag("", phase2));
   } else {
     stripCPEToken_ = c.consumes(edm::ESInputTag("", p.getParameter<std::string>("StripCPE")));
+    if (pixelQualityFlags_ != 0) {
+      pixelCablingToken_ = c.consumes();
```

Comment:
I am not particularly fond of this pattern in which consuming the pixel phase-1 FED cabling map is bound to the fact that there is someone in the job that requires the phase-1 SiStrip CPE, though at least from a logical point of view the two conditions should always be synched.